networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/Eth2NetworkBuilder.java

```diff
@@ -151,6 +154,13 @@ public Eth2NetworkBuilder recentChainData(final RecentChainData recentChainData)
     return this;
   }
 
+  public Eth2NetworkBuilder upstreamAttestationPipe(
```

Comment:
nit: Probably could rename the method to `attestationListener` or `attestationConsumer`.

Reply:
Done.